charts/tigera-operator/Makefile

```diff
@@ -0,0 +1,23 @@
+include ../../metadata.mk
+include ../../lib.Makefile
+
+OPERATOR_REPO ?= tigera/operator
+OPERATOR_VERSION ?= $(shell grep '^ *version:' values.yaml | sed -e 's/.*version: //')
+
+TMP_DIR = .tmp
```

Comment:
You might want to use `mktemp` for this

Reply:
I think it should be consistent. If there were more targets that used that the directory then it will be better for that to be a specific directory. Would you prefer if I called it "build" or something like that?

Reply:
Consistent with what? `mktemp` makes a temporary directory in `/tmp` - e.g., `/tmp/tmp.Awllr9y9VR`

I'm just suggesting that the temporary dir doesn't need to be within the git tree